script.js

```diff
@@ -135,11 +135,27 @@ function initListeners() {
     if (target.classList.contains("options__trash")) {
       let str = target.closest(".notes__item");
       str = str.querySelector(".notes__descr").textContent;
-      console.log(str);
+      // console.log(str);
       let updatedObj = notesObj.filter((note) => note.note !== str);
       notesObj = updatedObj;
     }
 
+    // check note
+    if (
+      target.classList.contains("notes__item-check") ||
+      target.tagName === "IMG"
+    ) {
+      let str = target.closest(".notes__item");
```

Comment:
Problem 2: Repeating the same code
In your code, you're repeating this:

```js
target.closest(".notes__item").querySelector(".notes__descr").textContent;
```

Why it's bad:
Repeating code makes it harder to fix or change later.
It’s harder to read.

Better way:
Use variables to store those values once:

```js
const noteElement = target.closest(".notes__item");
const noteText = noteElement.querySelector(".notes__descr").textContent;
```

Now you can reuse noteText wherever needed.